cocos/game/splash-screen.ts

```diff
@@ -536,6 +536,8 @@ export class SplashScreen {
                     );
 
                     cmdBuff.bindPipelineState(bgPso);
+                    // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+                    cmdBuff.bindDescriptorSet(SetIndex.GLOBAL, pipeline.descriptorSet);
```

Comment:
Is `pipeline` a global variable? Why is it a `any` value here? Could we import it from somewhere to avoid to use `eslint-disable-next-line`?